### PR TITLE
Strings to bundle

### PR DESCRIPTION
**Pull request by mlpozdeev**
Added message strings to created bundle

### CODE REVIEW

File: src/main/kotlin/ru/ozon/ideplugin/kelp/KelpConfig.kt [hunk 1]
```diff
@@ -67,12 +67,12 @@ class KelpConfig(
...
-        val functionSimpleNamePrefix: String? = null,
-        val appPackageName: String,
-        val componentDeeplink: String,
-        val intentionName: String = "🚀 Open in design system demo app",
-        val apkInstalling: ApkInstalling? = null,
+            val functionFqnPrefix: String,
+            val functionSimpleNamePrefix: String? = null,
+            val appPackageName: String,
+            val componentDeeplink: String,
+            val intentionName: String = KelpBundle.message("openInDemoAppIntentionName", "\uD83D\uDE80"),
```
Comment (popovanton0) [inline src/main/kotlin/ru/ozon/ideplugin/kelp/KelpConfig.kt]: Is it needed to pass an emoji as a parameter? Maybe bake it into the string in `.properties` file?

Comment (mlpozdeev) [reply src/main/kotlin/ru/ozon/ideplugin/kelp/KelpConfig.kt]: There is problem with encoding, not working like that

File: src/main/resources/messages/KelpBundle.properties [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+demoAppLatestVersionNotFoundErrorMessage=Latest version of the demo app wasn't found by ths regex '{0}' in this file {1}
+apkFileNotFoundErrorMessage=Demo apk file was not found here: {0}
+openedNotificationContent={0} opened in demo app
+componentOpeningErrorTitle=Error While Opening DS Component
+noSuchComponentNotificationContent=Couldn't determine the fully qualified name of the design system component
+noRunningDevicesNotificationTitle=No running devices found
+noRunningDevicesNotificationContent=Please, start one, or wait for it to finish loading
+noAndroidSdkNotificationContent=No android sdk found. Please, install it
+
+openInDemoAppIntentionName={0} Open in design system demo app
```
Comment (popovanton0) [inline src/main/resources/messages/KelpBundle.properties:19]:
```suggestion
openInDemoAppIntentionName=\uD83D\uDE80 Open in design system demo app
```
This works. Used this action to paste as unicode:

Screenshot 2024-02-02 at 12 03 10

File: src/main/resources/messages/KelpBundle.properties [hunk 3]
```diff
@@ -0,0 +1,22 @@
+kelpIntentionsFamilyName=Kelp intentions
+kelpNotificationGroup=Kelp
+openingInDemoAppMessage=Opening in demo app
+turningOnScreenMessage=Turning the screen on
+chooseDevicePopupTitle=Choose a Device
+checkingInstalledAppVersionMessage=Checking installed demo app version
+uninstallingPreviousAppMessage=Uninstalling previous demo app version
+installingAppMessage=Installing demo app apk
+demoAppLatestVersionInfoFileNotFoundErrorMessage=File that contains info about latest version of the demo app wasn't found: {0}
+demoAppLatestVersionNotFoundErrorMessage=Latest version of the demo app wasn't found by ths regex '{0}' in this file {1}
```
Comment (popovanton0) [inline src/main/resources/messages/KelpBundle.properties:10]:
```suggestion
demoAppLatestVersionNotFoundErrorMessage=Latest version of the demo app wasn't found by this regex '{0}' in this file {1}
```

**Review by kollesnica-power — APPROVED** [review]

**Review by popovanton0 — APPROVED** [review]

**Pull request merged**